msteams-platform/concepts/build-and-test/prepare-your-o365-tenant.md

```diff
@@ -30,6 +30,9 @@ Enable Teams for your organization and for more information, see [enabling Teams
 
 ## Enable custom Teams apps and turn on custom app uploading
 
+> [!IMPORTANT]
+> You can skip the steps below if you chose the Instant Sandbox option with the Teams sample data pack which will automatically enable the **Upload custom apps** setting.
```

Comment:
Suggested revision: "You do not have to follow the steps to enable sideloading if you have an instant sandbox or a configurable sandbox with the Teams sample data pack installed. The **Upload custom apps** setting is enabled by default in the Teams sample data pack."

Reply:
@Lauragra 

We have updated the note in the article as per your suggestion. Request you to review the changes and approve.

![image](https://user-images.githubusercontent.com/93579821/214105931-586dda2f-d36d-4497-9fd8-60525499268a.png)